src/Rule.js

```diff
@@ -37,11 +37,29 @@ module.exports = class Rule extends ChainedMap {
   }
 
   toConfig() {
+    const useEntries = this.uses.entries() || {};
+    const names = Object.keys(useEntries);
+    const uses = [...names];
+
+    names.forEach(name => {
+      const { _before, _after } = useEntries[name];
+
```

Comment:
Do we need a check to ensure both `_before` and `_after` are not set for the same rule, and fail loudly if so? (And a mention in the docs?) Since currently if both are set, the `_after` clobbers the splices made by the `_before` handling.